api/end_device.proto

```diff
@@ -185,10 +189,10 @@ message MACSettings {
   DataRateIndexValue ping_slot_data_rate_index = 3;
   // Frequency of the class B ping slot (Hz).
   // If unset, the default value from Network Server configuration will be used.
-  google.protobuf.UInt64Value ping_slot_frequency = 4 [(validate.rules).uint64.gte = 100000];
+  FrequencyValue ping_slot_frequency = 4;
```

Comment:
Doesn't this break API compatibility? A `google.protobuf.UInt64Value` renders to JSON as a number (possibly wrapped in a string). A `FrequencyValue` seems to render to JSON as an object.